library/options.inc.php

```diff
@@ -184,7 +184,7 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
 			
 				$optionValue = attr($lrow ['option_id']);
 			
-				if ($multiple && (strlen ( $currvalue ) == 0 && $lrow_backup ['is_default']) || 
```

Comment:
Note that this won't change anything here since this code is only use if $multiple per line 174
(that being said, this $multiple is redundant and can be removed since dealt with above in line 174)